static/js/popovers.js

```diff
@@ -170,13 +244,25 @@ function message_info_popped() {
     return current_message_info_popover_elem !== undefined;
 }
 
+function reaction_popped() {
+    return current_message_reactions_popover_elem !== undefined;
+}
+
 exports.hide_message_info_popover = function () {
     if (message_info_popped()) {
         current_message_info_popover_elem.popover("destroy");
         current_message_info_popover_elem = undefined;
     }
 };
 
+exports.hide_reactions_popover = function () {
+    if (reaction_popped()) {
+        $('.popover').remove();
```

Comment:
I think we can just delete this, but only after fixing the fact that the  `current_message_reactions_popover_elem` gets deleted once you unhover the message.